docs/DeveloperGuide.md

```diff
@@ -486,367 +498,250 @@ Hence, any modification to an employee after SudoHR is initialized from storage
 
-**Use case: UC14 - Add employee to event**
-**MSS:**
-1. User requests to __list events(UC6)__
-2. SudoHR shows all events and their details
-3. User requests to list employees
-4. SudoHR shows all employees and their details
-5. User requests to add employee to event on SudoHR
-6. SudoHR adds employee to event.
-7. Use case ends.
+   Use case ends.
 
 **Extensions**
 
-* 2a. The event list is empty.
-
-  Use case ends.
-
-* 4a. The employee list is empty.
-
-  Use case ends.
-
-* 5a. The given Employeeindex is invalid.
+- 1a. The given department does not exist.
 
-    * 5a1. SudoHR shows an error message.
+  - 1a1. SudoHr shows an error message.
 
-      Use case resumes at step 4.
+    Use case resumes at step 1.
 
-* 5b. The given Eventindex is invalid.
+- 1b. There are no employees in the given department.
 
-    * 5b1. SudoHR shows an error message.
+  Use case ends.
 
-      Use case resumes at step 4.
+**Use case: Listing all departments that an employee is in**
 
-**Use case: UC15 - List employees in an event**
-**MSS:**
-1. User requests to __list events(UC6)__
-2. SudoHR shows all events and their details
-3. User requests to __list employees in a event(UC8)__
-4. SudoHR shows all employees added to an event and their details
-5. Use case ends.
+1. User requests to list all the departments that a given employee is in.
+2. SudoHr lists all the department that the given employee is in.
 
 **Extensions**
 
-* 2a. The event list is empty.
+- 1a. The given employee does not exist.
+  - 1a1. SudoHr shows an error message.
+- 1b. The employee is not in any department.
 
   Use case ends.
 
-* 3a. The given Index is invalid.
-
-    * 3a1. SudoHR shows an error message.
+**Use case: list the department headcount**
 
-      Use case resumes at step 4.
+1. User requests to list all the employees present in a given department.
+2. SudoHr lists all the employees present in the given department for the current date.
 
-**Use case: UC16 - Delete employee from event**
-**MSS:**
-1. User requests to __list events(UC6)__
-2. SudoHR shows all events and their details
-3. User requests to __list employees in a event(UC8)__
-4. SudoHR shows all employees added to an event and their details
-5. User requests to delete employee to event on SudoHR
-6. SudoHR delete employee from event.
-7. Use case ends.
+   Use case ends.
 
 **Extensions**
 
-* 2a. The event list is empty.
+- 1a. User requests to list employees present on a date that is before the current date.
 
-  Use case ends.
-
-* 4a. The employee list is empty.
+  - 1a1. SudoHr shows an error message.
 
-  Use case ends.
+- 1b. User requests to list employees present on a date that is not within a year from the current date.
 
-* 5a. The given Employeeindex is invalid.
+  - 1b1. SudoHr shows an error message.
 
-    * 5a1. SudoHR shows an error message.
+- 1c. User requests to list employees present on a specified that is after the current date but within a year from the current date.
```

Comment:
Type at "on a specified"

Reply:
thanks for catching this! btw your type is a type also XD